waterdata/utils.py

```diff
@@ -0,0 +1,49 @@
+from io import StringIO
+from urllib.parse import urljoin
+
+import pandas as pd
+import requests as r
+
+
+def get_water_services_data(hostname, url_path):
```

Comment:
Since we are using requests, it might make sense to pass the path in as path along with a params dictionary